### ISSUE TITLE
[HOLD for payment 2023-07-05] [$1000] FEATURE REQUEST: Increase the size of composer a bit bigger and easier to tap

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#15819]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Problem:
The size of the composer is smaller than standard button size

## Solution:
Increase the size of our composer just slightly so that the send button becomes 40px tall, which is a standard button size and is a bit bigger and easier to tap

## Context/Examples/Screenshots/Notes:
<img width="1024" alt="image (2)" src="https://user-images.githubusercontent.com/43996225/224220866-fe6a4bf0-c15e-45f8-9530-433b60cf289f.png">

**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @shawnborton 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1678380227758669

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0114ab49b58fd19b29</li>
        <li>Upwork Job ID: 1669463781749805056</li>
        <li>Last Price Increase: 2023-06-15</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
This issue has not been updated in over ***15*** days.  eroding to ***Monthly*** issue.

P.S. Is everyone reading this sure this is really a near-term priority? Be brave: if you disagree, go ahead and close it out. If someone disagrees, they'll reopen it, and if they don't: one less thing to do!

**Comment by shawnborton** [comment]
I feel like we still want to do this. Do you agree @JmillsExpensify @trjExpensify ? I think I'll slap External on it so we can get this through the process.

**Comment by syedsaroshfarrukhdot** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Increase the size of composer a bit bigger and easier to tap

### What is the root cause of that problem?

componentSizeNormal variable value is set to `40px`

### What changes do you think we should make in order to solve the problem?


componentSizeNormal variable value is set to `40px` so for composer we can redefine a new variable with value `48px` and use it to style composer

Furthermore, we will proceed with implementing the required modifications to the styles.js file, specifically for components such as chatItemSubmitButton and chatItemEmojiButton.

Additionally, if necessary, we can also consider adjusting the width and height of the emoji button to enhance its usability

### What alternative solutions did you explore? (Optional)

**Reminder:** Please use plain English, be brief and avoid jargon. Feel free to use images, charts or pseudo-code if necessary. Do not post large multi-line diffs or write walls of text. Do not create PRs unless you have been hired for this job.

<!---
ATTN: Contributor+

You are the first line of defense in making sure every proposal has a clear and easily understood problem with a "root cause". Do not approve any proposals that lack a satisfying explanation to the first two prompts. It is CRITICALLY important that we understand the root cause at a minimum even if the solution doesn't directly address it. When we avoid this step we can end up solving the wrong problems entirely or just writing hacks and workarounds.

Instructions for how to review a proposal:

1. Address each contributor proposal one at a time and address each part of the question one at a time e.g. if a solution looks acceptable, but the stated problem is not clear then you should provide feedback and make suggestions to improve each prompt before moving on to the next. Avoid responding to all sections of a proposal at once. Move from one question to the next each time asking the contributor to "Please update your original proposal and tag me again when it's ready for review".

2. Limit excessive conversation and moderate issues to keep them on track. If someone is doing any of the following things please kindly and humbly course-correct them:

- Posting PRs.
- Posting large multi-line diffs (this is basically a PR).
- Skipping any of the required questions.
- Not using the proposal template at all.
- Suggesting that an existing issue is related to the current issue before a problem or root cause has been established.

**Comment by trjExpensify** [comment]
Yeah, nice! Much easier to smash that send button!

**Comment by MariaHCD** [comment]
Not overdue, @daraksha-dk was just assigned.

**Comment by zanyrenney** [comment]
Awesome, thanks @daraksha-dk

**Comment by zanyrenney** [comment]
bump @sobitneupane and @MariaHCD on reviewing the PR, any updates here?

**Comment by zanyrenney** [comment]
I see this is labeled "not a priority" should we make this weekly not daily?

**Comment by zanyrenney** [comment]
I am heading OOO shortly, so assigning another bug0 team member to help watch over this issue whilst I am OOO! Thanks!

**Comment by shawnborton** [comment]
Tested quickly and I think this feels pretty good to me.

**Comment by isabelastisser** [comment]
I am unassigning myself because I'm going OOO until July 5. @zanyrenney is the original assignee, so I will leave her assigned to the issue.

**Comment by joekaufmanexpensify** [comment]
I was assigned [this bug ](https://github.com/Expensify/App/issues/21692). Were we changing the size of the FAB button in this issue to be 32x40? 

I see in the tests for the [PR here](https://github.com/Expensify/App/pull/21131#issue-1765632775) that this is the expected size, but curious if we meant to change that in this issue.

**Comment by shawnborton** [comment]
I don't think that's a bug, I think it's fine as-is.

**Comment by joekaufmanexpensify** [comment]
Cool cool, ty!

**Comment by zanyrenney** [comment]
@sobitneupane and @daraksha-dk I have invited you to the job on upwork. there is no reporting bonus here as this was reported internally. 

Please accept the job for payouts.

**Comment by sobitneupane** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:


This is a feature request not a bug. So, I don't think BugZero Checklist applies here.

**Comment by sobitneupane** [comment]
Requested money on Expensify.

**Comment by zanyrenney** [comment]
Brilliant thank you @sobitneupane - I'll view your payment as complete then!

**Comment by zanyrenney** [comment]
@daraksha-dk can you confirm if you'll be doing the same, and requesting money on Expensify instead?

**Comment by sobitneupane** [comment]
@zanyrenney @daraksha-dk It is only being used (internally) by few Contributor+ now. More context: https://expensify.slack.com/archives/C02NK2DQWUX/p1686871517189609

**Comment by zanyrenney** [comment]
hey @daraksha-dk offer sent, please accept.

[no bonus due](https://github.com/Expensify/App/issues/15819#issuecomment-1607580264)

**Comment by zanyrenney** [comment]
Paid @daraksha-dk 

@sobitneupane will be managing payment through Expensify.

there is no reporting bonus as it was an internal report. 

Good to close here!
